roles/gitops_configure_repo/tasks/main.yaml

```diff
@@ -12,11 +12,14 @@
         app.kubernetes.io/part-of: argocd
       data:
         ssh_known_hosts: |
-          {{ gitlab_ssh_known_hosts }}
+          {{ ssh_known_hosts }}
+  when: ssh_known_hosts is defined
+  no_log: true
 
 - name: Register key
-  ansible.builtin.command: cat "{{ gitlab_key_path }}"
-  register: gitlab_key
+  ansible.builtin.command: cat "{{ ssh_key_path }}"
```

Comment:
I feel like slurp should be used rather than cat here but that's personal preference.

Reply:
Even better, using 
```
ansible.builtin.set_fact:
  ssh_key:"{{ lookup('file', ssh_key_path) }}"
```
and it save a task

Reply:
If its on the runner yes :)

Reply:
Thanks for the comments.
I updated the way we consume the ssh key.
Regarding the name of the role, I was planning on keeping it aligned with the origin role in the eco-validation repository, So for now I'd rather not rename it.